tools/walletextension/api/staticOG/style.css

```diff
@@ -35,11 +203,13 @@ input { display: table-cell; }
 
 /* basic style for the accounts table*/
 table {
```

Comment:
Maybe center the table and use same style as the rest of the webpage:

<img width="993" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/4050716/7df68e15-6a45-4e8c-a744-012b005fde1d">

In the table we display all the accounts that are connected to metamask (for this URL), even if they are not connected to Obscuro (connected = False), but we don't give an option to the user to add them. Is this ok?

Reply:
It's not ideal, but it's okay for this first version. I'll raise a ticket for it.